src/flexmeasures_client/client.py

```diff
@@ -85,6 +86,12 @@ def __post_init__(self):
             )
         if len(self.password) < 1:
             raise EmptyPasswordError("password cannot be empty")
+        parts = self.host.split(':')
```

Comment:
Deserves a comment, like "if port came as part of host, separate it"

Reply:
I do not much like comments, as they tend to go out-of-sync with the code.
What do you think about splitting it into a function?
Do you use some convention for private functions?

Reply:
To me, the comment is in style with the block above, which also has one.

A function also works.

Or a less specific comment.

Reply:
Updated the code